src/Controllers/Admin/AppSettingsController.php

```diff
@@ -413,4 +463,62 @@ public function delete_app( $slug )
 
         return $response;
     }
+
+    /** Soft delete an app by its slug.
+     * This function marks an app as soft deleted based on its slug.
+     *  @param string $slug The slug of the app to be soft deleted.
+     * @return \Symfony\Component\HttpFoundation\RedirectResponse Redirects to the admin page with a success message.
+     */
+     public function softDelete_app( $slug)
```

Comment:
@Muniraj132 , small code style issue here, this should be either softdelete_app, or soft_delete_app according to WordPress code style.

Reply:
Also, please add a space between $slug and )

Reply:
@incraigulous Thank you for pointing that out. I'll update the code to follow the WordPress coding standards, changing it to either softdelete_app or soft_delete_app as suggested. I'll also add the space between $slug and ).